administration/src/bp-modules/cards/hooks/useCardGenerator.ts

```diff
@@ -59,6 +66,23 @@ const useCardGenerator = ({ region, initializeCards = true }: UseCardGeneratorPr
   const [createCardsMutation] = useCreateCardsMutation()
   const [deleteCardsMutation] = useDeleteCardsMutation()
   const appToaster = useAppToaster()
+  const { t } = useTranslation('cards')
+
+  const [sendToFreinet] = useSendApplicationDataToFreinetMutation({
+    onCompleted: data => {
+      if (data.sendApplicationDataToFreinet === true) {
+        appToaster?.show({ intent: 'success', message: t('freinetDataSyncSuccessMessage') })
+      } else {
+        appToaster?.show({ intent: 'danger', message: t('errors:freinetDataSyncError') })
```

Comment:
❓ in which case is this needed?
I think if there is no error but the data was not send because its not activated for the region, there shouldn't be any toast

Reply:
Aha I get what you mean... just error for exceptions and ignore false return.. right?